src/styles/base.js

```diff
@@ -258,10 +267,17 @@ export const sliderBarHeight = 4;
 export const sliderHandleHeight = 12;
 export const sliderHandleWidth = 12;
 export const sliderHandleColor = '#D3D8E0';
+export const sliderHandleTextColor = 'black';
```

Comment:
why `black` it is coming out of nowhere? make it a same as `sliderHandleColor` since you are not showing anything

Reply:
done